controllers/comments.controllers.js

```diff
@@ -12,4 +11,17 @@ exports.getCommentsByArticleId = (request, response, next) => {
     .catch((err) => {
         next(err)
     })
+}
+
+
+exports.postComment = (request, response, next) => {
+    const {username, body} = request.body;
+    const {article_id} = request.params;
+
+    insertNewComment(article_id, username, body).then((Newcomment)=> {
```

Comment:
As per above this should be newComment or just simply comment as the variable name 🙂

Reply:
It's fixed now